app/views/katello/api/v2/subscription_facet/base.json.rabl

```diff
@@ -3,3 +3,7 @@ attributes :id, :uuid, :last_checkin, :service_level, :release_version, :autohea
 child :user => :user do
   attributes :id, :login
 end
+
+node(:facts) do |facet|
+  facet.select_facts
+end
```

Comment:
testing on a user's db, this seems to increase requests to the hosts api by 50%.  I'm not sure that is acceptable right now :(     We might need to discuss alternatives to this piece